common/src/main/java/us/myles/ViaVersion/protocols/protocol1_13to1_12_2/blockconnections/BasicFenceConnectionHandler.java

```diff
@@ -1,17 +1,22 @@
 package us.myles.ViaVersion.protocols.protocol1_13to1_12_2.blockconnections;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class BasicFenceConnectionHandler extends AbstractFenceConnectionHandler {
 
-    static void init() {
-        new BasicFenceConnectionHandler("fenceConnections", "minecraft:oak_fence");
-        new BasicFenceConnectionHandler("fenceConnections", "minecraft:birch_fence");
-        new BasicFenceConnectionHandler("fenceConnections", "minecraft:jungle_fence");
-        new BasicFenceConnectionHandler("fenceConnections", "minecraft:dark_oak_fence");
-        new BasicFenceConnectionHandler("fenceConnections", "minecraft:acacia_fence");
-        new BasicFenceConnectionHandler("fenceConnections", "minecraft:spruce_fence");
+    static List<ConnectionData.ConnectorInitAction> init() {
```

Comment:
What about using `Set` instead of `List` here? To me it looks like the elements don't have to be ordered, but maybe I'm wrong. :thinking:

Reply:
https://stackoverflow.com/questions/10799417/performance-and-memory-allocation-comparison-between-list-and-set As far as I know when only iterating using a list is enough (i.e. set also has their hashmap in the background)